lib/hal/hal_impression_ac073tc1a.ex

```diff
@@ -0,0 +1,309 @@
+defmodule Inky.HAL.ImpressionAC073TC1A do
+  @default_io_mod Inky.IO.Impression
+
+  @moduledoc """
+  An `Inky.HAL` implementation responsible for sending commands to the Inky
+  screen. It delegates to whatever IO module its user provides at init, but
+  defaults to #{inspect(@default_io_mod)}
+  """
+
+  @behaviour Inky.HAL
+
+  @color_map %{black: 0, white: 1, green: 2, blue: 3, red: 4, yellow: 5, orange: 6, miss: 1}
+  # @colors %{
+  #   :black => 0,
+  #   :white => 1,
+  #   :green => 2,
+  #   :blue => 3,
+  #   :red => 4,
+  #   :yellow => 5,
+  #   :orange => 6,
+  #   # Not a color, but is used to clear the display
+  #   :clear => 7
+  # }
+
+  # PANEL SETTING
+  @psr 0x00
+  # POWER SETTING
+  @pwr 0x01
+  # POWER OFF
+  @pof 0x02
+  # POWER OFF SEQUENCE SETTING
+  @pofs 0x03
+  # POWER ON
+  @pon 0x04
+
+  # BTST1
+  @btst1 0x05
+  @btst2 0x06
+  # @dslp 0x07
```

Comment:
It's above `@btst3` because that's the order it comes numerically. I'd like to keep the commented out `@dslp` because I got the value from the datasheet (at least I'm 90% I did :sweat_smile:)